public/css/homepage.css

```diff
@@ -104,38 +104,97 @@ body {
 #filters {
   display: flex;
   flex-flow: row;
-  justify-content: left;
+  justify-content: space-between;
 }
 
 #filters > div {
   margin: 0 1em;
+  border-bottom: var(--border-bottom-grey);
 }
 
-#filter-category-dropdown, #filter-location-dropdown {
-  background-color: white;
-  margin: 0 18px;
-  width: 100px;
+#filter-category, #filter-distance, #filter-duration, #filter-emergency {
+  font-size: 0.8em;
+  width: 170px;
+}
+
+#filter-category {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+}
+
+#category-label, #max-distance-label, #max-duration-label, #max-emergency-label {
+  width: 100%;
+  padding: 5px 0;
+  text-align: center;
+  color: var(--basic-white);
+  background-color: var(--accent-color);
+}
+
+#filter-category-dropdown {
```

Comment:
I am not sure if the user can see without arrows, that this selection is a dropdown.